package.json

```diff
@@ -11,8 +11,8 @@
     "classnames": "2.2.5",
     "fancy-dedupe": "0.1.0",
     "get-youtube-id": "1.0.0",
-    "history": "2.1.2",
-    "lodash.debounce": "4.0.7",
+    "history": "3.0.0",
```

Comment:
This change will fail due to needing an update of react-router which hasn't been published yet (https://github.com/reactjs/react-router/issues/3611). I'll update the branch once the first pass of tests have run.